runtime/lmsig_ksi-ls12.c

```diff
@@ -168,6 +169,14 @@ SetCnfParam(void *pT, struct nvlst *lst)
 			} else {
 				rsksiSetBlockTimeLimit(pThis->ctx, pvals[i].val.d.n);
 			}
+		} else if (!strcmp(pblk.descr[i].name, "sig.confinterval")) {
+			if (pvals[i].val.d.n < 0) {
+				LogError(0, RS_RET_ERR, "sig.confinterval "
+					"%llu invalid - signing disabled", pvals[i].val.d.n);
+				pThis->ctx->disabled = true;
```

Comment:
This miscofiguration does not appear to be that severe to disable log signing - perhaps use a default value instead and log a warning. But if this is a common approach I guess this is also fine.